TM1py/Services/ElementService.py

```diff
@@ -652,6 +713,15 @@ def get_parents(self, dimension_name: str, hierarchy_name: str, element_name: st
 
         return [record["Name"] for record in response.json()["value"]]
 
+    def get_parents_all_elements(self, dimension_name: str, hierarchy_name: str, **kwargs) -> List[str]:
```

Comment:
the type hint for the return value should be Dict[str, str] I think

Reply:
Nice function. Definitely useful. 

Let's call it `get_parents_of_all_elements`. Then it's more in line with existing functions like `get_number_of_elements`, `get_attribute_of_elements`

Reply:
Thanks for the pointer on the return value, fixed with https://github.com/cubewise-code/tm1py/pull/792/commits/6d66c211c833443f9d3f3500e986716e31b6242a

Good call, function renamed with https://github.com/cubewise-code/tm1py/pull/792/commits/6d66c211c833443f9d3f3500e986716e31b6242a and test case with https://github.com/cubewise-code/tm1py/pull/792/commits/0fe40b946a19ffadb11b91a29afcce275966f269